### PR TITLE
common.json: fix typos, minor improvements

### PR DESCRIPTION
**Pull request by kbdharun**

### CODE REVIEW

File: frontend/public/locales/en/common.json [hunk 1]
```diff
@@ -139,7 +139,7 @@
   "other-apps-by-developer": "Other apps by {{developer}}",
   "other-apps-by-projectgroup": "Other apps in the {{projectgroup}} group",
   "loading": "Loading…",
-  "could-not-find-match-for-search": "Could not find a match for search.",
+  "could-not-find-match-for-search": "Could not find a match for the search term.",
```
Comment (wjt) [inline frontend/public/locales/en/common.json:142]: This is an improvement IMO but "search term" seems a bit like jargon (and what if there is more than one term?)

How about:

> Could not find any apps matching your search.

**Review by wjt — COMMENTED** [review]
Just one suggestion :

**Comment by razzeee** [comment]
This seems to need a rebase

**Comment by kbdharun** [comment]
> This seems to need a rebase

Rebase and force push? (Wouldn't Squash & Merge work?)

**Pull request merged**